runtime/src/lib.rs

```diff
@@ -21,6 +21,7 @@
 #![no_std]
 #![warn(unsafe_op_in_unsafe_fn)]
 
+pub use libtock_platform;
```

Comment:
I don't think we should export the entire `libtock_platform` crate. Instead, I suggest you export the following function (probably from the `startup` module, which would need to be made `pub`):

```rust
pub fn handle_main_return<T: Termination>(result: T) -> ! {
    Termination::complete::<TockSyscalls>(result)
}
```

Then call `$crate::startup::handle_main_return($name())` in `libtock_unsafe_main()`.

Reply:
Done.